sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -566,10 +568,30 @@ def _insert_all_rows(
             skipInvalidRows=skip_invalid_rows,
             # TODO(silviuc): Should have an option for ignoreUnknownValues?
             rows=rows))
+
+    resource = '//bigquery.googleapis.com/projects/%s/datasets/%s/tables/%s' % (
+        project_id, dataset_id, table_id)
```

Comment:
Maybe add the resource calculation to a helper method? So it can be reused later? (I imagine you'll add these metrics for other methods?)

Reply:
or a constant IDK : )